### PR TITLE
download pre-release version

### PR DESCRIPTION
**Pull request by jdidion**

### CODE REVIEW

File: test-dxcompiler-in-project/Makefile [hunk 1]
```diff
@@ -1,12 +1,18 @@
 # TODO: find a way to download from draft release URLs; the below works for final releases.
 # For draft releases, save jar manually to resources/home/dnanexus/dxCompiler.jar
-DXCOMPILER_JAR_URL=https://github.com/dnanexus/dxCompiler/releases/download/2.7.0/dxCompiler-2.7.0.jar
+VERSION=2.8.3
+DXCOMPILER_PRERELEASE_PATH=project-G7GFgjQ0X386z5ZF24vvyFzp:dxCompiler-${VERSION}.jar
```
Comment (r-i-v-a) [inline test-dxcompiler-in-project/Makefile:4]: Is there code somewhere that uploads the prerelease jar to this project? I don't see it. @commandlinegirl has also suggested that uploading the jar as a GitHub artefact (similar to what we do with snapshots) could be a workaround to be able to upload / download drafts. Any reason to prefer putting it on the platform vs. on GitHub? @jdidion  @commandlinegirl

File: test-dxcompiler-in-project/Makefile [hunk 2]
```diff
@@ -1,12 +1,18 @@
 # TODO: find a way to download from draft release URLs; the below works for final releases.
```
Comment (r-i-v-a) [inline test-dxcompiler-in-project/Makefile:1]: TODO comment can be removed when draft upload / download issue is solved.

**Review by r-i-v-a — CHANGES_REQUESTED** [review]
See comments, automate upload?

File: test-dxcompiler-in-project/Makefile (hunk 1 above)
Comment (jdidion) [reply test-dxcompiler-in-project/Makefile:4]: Nope, it's a manual process. I created a project on prod to upload pre-release versions for internal testing prior to release. I don't think we want to be making pre-release versions available publicly.

**Pull request closed without merging**